File: data/plotme.m
function plotme()

seq = load('sequential_time_data.csv');
mpi = load('mpi_time_data.csv');

seq_y = mean(seq, 1);
mpi_y = mean(mpi, 1);
x = 1:250;

clf;
hold on;
plot(x, seq_y, 'b');
plot(x, mpi_y, 'r');
legend('sequential', 'parallel')
xlabel('iterations')
ylabel('time')

File: data/mpi_v_seq_time_comparison/plotme.R
library(ggplot2)

mpi.df <- read.csv("mpi_time_data.csv", header=F)
mpi.df <- mpi.df[,1:250]

seq.df <- read.csv("sequential_time_data.csv", header=F)
seq.df <- seq.df[,1:250]

df <- data.frame(x=1:250, y1=rowMeans(t(mpi.df)), y2=rowMeans(t(seq.df)))
cairo_ps(file="plot.eps", width=7,height=7)
p <- (ggplot(df, aes(x, y = value, color = variable)) +
      geom_line(aes(y = y1, col = "Parallel")) +
      geom_line(aes(y = y2, col = "Sequential")) +
      xlab("Number of queries") + ylab("Seconds taken") +
      scale_fill_discrete(breaks=c("Sequential", "Parallel")))
plot(p)
dev.off()
